packages/orchestrator-ui-components/src/rtk/endpoints/formFields.ts

```diff
@@ -0,0 +1,60 @@
+import { ImsNode } from '@/components';
+import { ContactPerson } from '@/components/WfoForms/formFields/types';
+import { BaseQueryTypes, orchestratorApi } from '@/rtk';
+
+const LOCATION_CODES_ENDPOINT = 'surf/crm/location_codes';
+const CONTACT_PERSONS_ENDPOINT = 'surf/crm/contacts';
+const IMS_NODES_ENDPOINT = '/surf/ims/nodes';
+
+const formFieldsApi = orchestratorApi.injectEndpoints({
```

Comment:
I think they are tough. They are only used in formfields

Reply:
True, but it would still make more sense to group them by domain/data type. Similar is currently the case for `subscriptions` queries. We have 2 queries in there. One for the SubscriptionListPage and one for the SummaryCards on the home page.